src/main/webapp/i18n/de/metis.json

```diff
@@ -27,17 +29,32 @@
             "searchEmoji": "Emoji suchen",
             "courseEmojiSelectionCategory": "Ausgewählte Emojis für diesen Kurs",
             "discussion": {
-                "header": "Diskussion",
+                "label": "Diskussion",
                 "show": "Diskussion anzeigen",
                 "hide": "Diskussion einklappen"
             },
             "overview": {
                 "title": "Beiträge",
-                "noPosts": "Es gibt noch keine Beiträge in diesem Kurs.",
-                "showApproved": "Zeige Beiträge mit bestätigten Antworten an",
-                "hideApproved": "Ignoriere Beiträge mit bestätigten Antworten"
+                "searchBarDefault": "Tippe hier, um nach einem bestimmten Beitrag zu suchen...",
```

Comment:
```suggestion
                "searchBarDefault": "Klicke hier, um nach einem bestimmten Beitrag zu suchen...",
```
Since I don't think we should assume the user has a touchscreen

Reply:
That's a misunderstanding, It's typing in this context
<img width="1056" alt="Bildschirmfoto 2021-09-14 um 11 26 36" src="https://user-images.githubusercontent.com/41366522/133232407-1cbf042b-41e4-40e2-a6db-7c44bb680064.png">

Reply:
I see. That makes sense, although it is still a rather uncommon way of expressing it; All examples I found went something along the lines of "search..."

Reply:
Yes, that's true, I felt the same when reading it again.. I think about something else :)